docs/platforms/javascript/common/configuration/integrations/nodeprofiling.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: ProfilingIntegration
+description: "Collect & view performance insights for JavaScript programs with Sentry’s JavaScript Profiling integration."
```

Comment:
```suggestion
description: "Collect & view performance insights for JavaScript Node programs with Sentry’s Node Profiling integration."
```
I wonder if it's more accurate to call out that this is specifically for node programs rather than saying javascript? If it doesn't make sense, feel free to disregard this